content/using-atom/sections/moving-in-atom.md

```diff
@@ -40,6 +40,111 @@ You can also move directly to a specific line (and column) number with <kbd clas
 
 ![Go directly to a line](../../images/goto.png "Go directly to a line")
 
+
+#### Additional Movement and Selection Commands
+
+Atom also has a few movement and selection commands that don't have a keyboard shortcut mapped to it.   You can access these commands from the [Command Palatte](http://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette), but if you find yourself using a command that doesn't have a keyboard shortcut often, have no fear!  You can easily add a entry to your `keymaps.cson` to create a keyboard shortcut.  
+
+For example,  the command `editor:move-to-beginning-of-screen-line` is available in the command palette, but it's not binded to any keyboard shortcut.  To add create a keyboard shortcut you need to add a entry in your `keymaps.cson` file.   For `editor:select-to-previous-word-boundary`, you can add the following to your `keymaps.cson`:
```

Comment:
> binded

bound

Reply:
"To add create a keyboard shortcut" - should remove either "add" or "create".

Also same as above, "an entry" and "keymap.cson".